appyters/KEA3_Appyter/KEA3-Appyter.ipynb

```diff
@@ -169,95 +169,110 @@
     "    all_prePPI = []\n",
     "    all_PTMsigDB = []\n",
     "    \n",
-    "    for kinase in range(0, 10):\n",
+    "    # Length of all the libraries (for calculation of )\n",
+    "    max_ranks = {}\n",
+    "    for library in range(2,13):\n",
+    "        lib_name = (list(results.keys())[library])\n",
```

Comment:
This looks really strange and is hard to follow. Typically if you're using integer indexes instead of string ones you're doing something strange.

here you should just do:
```python
for lib_name in results.keys():
  # ...
```